modules/apps/27-interchain-accounts/controller/client/cli/tx.go

```diff
@@ -94,21 +92,12 @@ via {packet-timeout-timestamp}`),
 				}
 			}
 
-			timeoutHeightStr, err := cmd.Flags().GetString(flagPacketTimeoutHeight)
-			if err != nil {
-				return err
-			}
-			timeoutHeight, err := clienttypes.ParseHeight(timeoutHeightStr)
```

Comment:
I think we can remove the timeout height flag?

Reply:
@colin-axner I removed this flag, and also gave the remaining flag a more descriptive name. Please take a look at the changes when you have a minute!